tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -184,23 +266,52 @@ def increment(self, event, count, environment=None, timestamp=None):
             data["environment"] = environment
 
         for _ in range(count):
-            self.store_event(
-                data=data,
-                project_id=self.project.id,
-            )
+            self.incr_event(data, self.project.id, timestamp, pass_txn_timestamp)
+
+
+class ErrorEventFrequencyConditionTestCase(
```

Comment:
I probably made a huge mess of things while I attempted to rearrange stuff but here's my conundrum:

The `StandardIntervalMixin` is used by `EventFrequencyConditionTestCase` and `EventUniqueUserFrequencyConditionTestCase` presumably to not repeat code. I added `ErrorEventFrequencyConditionTestCase`, `PerformanceEventFrequencyConditionTestCase`, `ErrorEventUniqueUserFrequencyConditionTestCase`, and `PerformanceEventUniqueUserFrequencyConditionTestCase`, but I'm now not sure how to use the mixin. I'd need to copy/paste it and duplicate the tests so that it creates either an error event or a performance event. I'm also introducing a lot of duplication by redefining `incr_event` in multiple classes and it's the same code. 

I can do a factory method so that before `incr_event` is called I conditionally pick the class, but that felt hacky. Idk, I'm having a hard time reorganizing this huge test file.